backend/api/Database/Context/InitDb.cs

```diff
@@ -84,6 +86,7 @@ private static List<Deck> GetDecks()
             Id = Guid.NewGuid().ToString(),
             Plant = plants[0],
             Installation = plants[0].Installation,
+            DefaultLocalizationPose = null,
```

Comment:
We could set one to something usefull rather than "null", might be useful to have one for testing

Reply:
Agreed, but thought to keep it as null until we started using it